lib/babel.js

```diff
@@ -92,6 +93,11 @@ module.exports = (babel) => {
       t.memberExpression(id, t.identifier('setAttribute')),
       [t.stringLiteral(attr), value])
 
+  const setDomAttributeNS = (id, attr, value, ns = null) =>
+    t.callExpression(
+      t.memberExpression(id, t.identifier('setAttributeNS')),
+      [ns, t.stringLiteral(attr), value])
```

Comment:
I suspect the build is failing because `ns` is `null`; it should likely be a `t.nullLiteral()` instead

Reply:
Thank you. yep, sorry it's not finished yet.
I still study babel's cookbook, but found some more issues already.

May I ping as soon I think I'm finished.
I will also test it in my app, to see if they are working again.

Reply:
sure! feel free to ask if you run into anything too